### PR TITLE
Update DLS001.xml

### PR DESCRIPTION
**Pull request by Addisie**

### CODE REVIEW

File: Abennat/DLS001.xml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 schematypens="http://relaxng.org/ns/structure/1.0"?>
-<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="DLS001" xml:lang="en" type="mss">
     <teiHeader>
         <fileDesc>
             <titleStmt>
                 <title>Dǝggʷā</title>
+                <title>Dǝgʷā</title>
```
Comment (eu-genia) [inline Abennat/DLS001.xml:10]: dear Addisie, you do not need a second line for the title, line 9 (Dǝggʷā) was correct, I had already corrected it

File: Abennat/DLS001.xml [hunk 2]
```diff
@@ -51,7 +52,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             </langUsage>
         </profileDesc>
         <revisionDesc>
-            <change who="MA" when="2022-04-07">Created entity</change>
+            <change who="MA" when="2022-04-07">Created entity
```
Comment (eu-genia) [inline Abennat/DLS001.xml:55]: dear Addisie, now there is no closing </change> tag at the end of line 55 (it was there, but you deleted it)

File: Abennat/DLS001.xml [hunk 3]
```diff
@@ -51,7 +52,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+            <chnge who="Addisie" when="2022-04-15"></change>
```
Comment (eu-genia) [inline Abennat/DLS001.xml:56]: the tag is called "change" not chnge, and I believe you did not really change anything so I would not merge this pull request unless there are true changes

Did you check that your Atom works correctly? actually Atom should highlight errors and help you avoid them. Please follow the instructions in my last email.

**Comment by Addisie** [comment]
Dear Eugenia,
Thank you for your correction.
However, I was trying to remember to train again what we did before, the
document didn't need any change as you already did correctly.
Thus you don't need to merge it.
Regarding the Atom, I checked my desktop Athom, it is the 0.9 version and I
tried to downgrade it by following your instructions. Unfortunately when I
finished the process I didn't get any downgrade.
I also uninstalled the new (0.9) version and tried to download the old
version (0.8.4). I can't  do that as the only App on free software.com is
Atom 0.9
So if you recover well and have time for me, we can do that with you on
Thursday or Friday by the time that is appropriate to you.
Many thanks,
Addisie.

**Comment by eu-genia** [comment]
I am back to office so please come any time and I will try to fix the Atom problem for you.

**Comment by Addisie** [comment]
Thank you very much for your willingness to help me.

**Pull request closed without merging**